src/app/guides/page.tsx

```diff
@@ -1,320 +1,29 @@
-import { Orbit, Plus } from "lucide-react";
-import React from "react";
+"use client";
 
```

Comment:
To save yourself, pull from upstream and for this page, accept incoming changes because the page to be rendered at / already is here on the repo